src/ontology/uberon-edit.obo

```diff
@@ -224681,6 +224681,71 @@ relationship: dc-contributor https://orcid.org/0000-0001-7655-4833 ! Ellen Quard
+intersection_of: part_of UBERON:0002439 ! myenteric nerve plexus
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+property_value: dcterms-date "2025-02-13T17:17:15Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600119
+name: myenteric ganglion of small intestine
+def: "A myenteric ganglion located in the myenteric plexus of the small intestine." [ILX:0793735]
+xref: ILX:0793735
+intersection_of: UBERON:8600118 ! myenteric ganglion
+intersection_of: part_of UBERON:0002108 ! small intestine
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+property_value: dcterms-date "2025-02-13T17:20:16Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600120
+name: atrial intrinsic cardiac ganglion
+def: "A ganglion that is part of the atrial region of the heart. The activation of efferent neurons in this ganglion results in the modulation of heart rate, atrio-ventricular node conduction, and inotropism of atria and ventricles." [ILX:0793555, PMID:23559611, PMID:33255284]
+xref: ILX:0793555
+intersection_of: UBERON:0014463 ! cardiac ganglion
+intersection_of: part_of UBERON:0002081 ! cardiac atrium
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+property_value: dcterms-date "2025-02-13T17:23:00Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600121
+name: lumbar ganglion
+def: "A ganglion that is part of the lumbar portion of the sympathetic trunk. The neurons in a lumbar ganglion are postganglionic sympathetic neurons that synapse with axons from preganglionic sympathetic neurons." [https://www.elsevier.com/resources/anatomy/nervous-system/peripheral-nervous-system/lumbar-ganglia/23245, PMID:30969736, PMID:33085353, wikipediaversioned:Lumbar_ganglia&oldid=1023349088]
+comment: In humans there are four lumbar ganglia and they are positioned anterolateral to the lumbar vertebrae and deep to the psoas major muscle. {xref="PMID:30969736", xref="wikipediaversioned:Lumbar_ganglia&oldid=1023349088"}
+synonym: "lumbar paravertebral ganglion" EXACT [FMA:6472]
+synonym: "lumbar sympathetic ganglion" EXACT [FMA:6472]
+xref: FMA:6472
+is_a: UBERON:0001807 ! paravertebral ganglion
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-Barbé
+relationship: part_of UBERON:0005462 ! lower back
+property_value: dcterms-date "2025-02-13T17:25:41Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600122
+name: sacral ganglion
+def: "A ganglion that is part of the sacral portion of the sympathetic trunk. A sacral ganglium consists of postganglionic sympathetic neurons that receive presynaptic input from preganglionic sympathetic axons descending through the sympathetic trunk from thoracolumbar levels." [FMA:6473, https://www.elsevier.com/resources/anatomy/nervous-system/peripheral-nervous-system/sacral-ganglia/24872, PMID:33085353, wikipediaversioned:Sacral_ganglia&oldid=1222899024]
+xref: FMA:6473
+xref: sacral sympathetic ganglion {xref="FMA:6473"}
```

Comment:
This is not a valid cross-reference. Presumably the intention was that is should be synonym?